BAZProjectC1/Billboard/MovieCell.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  MovieCell.swift
+//  BAZProjectC1
+//
+//  Created by efloresco on 21/09/22.
+//
+
+import UIKit
+
+class MovieCell: UICollectionViewCell {
+    @IBOutlet weak var imgDetail: UIImageView!
+    @IBOutlet weak var lblTitle: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+    
+    func configureCellWithUrl(movieInfo: Movie) {
+        lblTitle.text = movieInfo.title
+        imgDetail.loadFrom(strUrl: "\(strPathImage)\(movieInfo.poster_path)")
```

Comment:
Qué te parece si movemos esta lógica al modelo.
La UI debería limitarse lo más posible a solo asignar datos.

Podrías hacer algo así:

```
struct Movie {
   var imageDetail: String {
       "\(strPathImage)\(poster_path)")
   }
}
```
Y después solo lo reemplazas así:
```suggestion
        imgDetail.loadFrom(strUrl: movieInfo.imageDetail)
```

Reply:
resuelto